### PR TITLE
genz-117:Solution for palindrome structure

### PR DESCRIPTION
**Pull request by sarah-alshukaily**

### CODE REVIEW

**Review by Rudainasaleh — APPROVED** [review]

**Review by Sughiya-AlSaid — APPROVED** [review]

**Review by Abbas-Abdulrab — DISMISSED** [review]

**Review by aljab017 — APPROVED** [review]

File: examples/sara_work_area/genz_117_pali_structure.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+value = int(input("Please enter the number of lines: "))
```
Comment (CodelineAtyab) [inline examples/sara_work_area/genz_117_pali_structure.py:1]: Lets add mutiline string 
"""
pali_structure.py says: What is my responsibility
"""

Comment (CodelineAtyab) [inline examples/sara_work_area/genz_117_pali_structure.py:1]: Meaningful variable names

File: examples/sara_work_area/genz_117_pali_structure.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+def pali_structure(value):
+    x=""
```
Comment (CodelineAtyab) [inline examples/sara_work_area/genz_117_pali_structure.py:4]: Consider changing the name

**Review by CodelineAtyab — CHANGES_REQUESTED** [review]
Please respond to comments

File: examples/sara_work_area/genz_117_pali_structure.py (hunk 2 above)
Comment (sarah-alshukaily) [reply examples/sara_work_area/genz_117_pali_structure.py:4]: Done

File: examples/sara_work_area/genz_117_pali_structure.py (hunk 1 above)
Comment (sarah-alshukaily) [reply examples/sara_work_area/genz_117_pali_structure.py:1]: Changed it to number of lines (no_lines)

**Review by aljab017 — APPROVED** [review]

**Review by Abbas-Abdulrab — APPROVED** [review]

**Review by Rudainasaleh — APPROVED** [review]

**Review by Alhasan-AJ — APPROVED** [review]

**Review by CodelineAtyab — APPROVED** [review]
Great Work!

**Pull request merged**